src/Components/Inputs/Input.scss

```diff
@@ -101,6 +102,38 @@
   color: $cf-input-text--focused;
 }
 
+.cf-input-clear-btn {
+  right: 4px;
+  top: 4px;
+  background: inherit;
+  border: none;
+  font-size: 1.9em;
+
+  &:focus:after {
+    opacity: 0;
+  }
+
+  &.cf-dismiss-button {
+    transform: none;
+    top: 4px;
+    right: 4px;
+  }
+
+  &.large {
```

Comment:
I believe the possible sizes of inputs are :
- extra small,
- small,
-medium
- large , 

I think you've only accounted for small and large. Try a medium or extra small input on your storybook, I imagine the X won't scale for those.

Reply:
i only did small (default) and large since those are the only use cases we need for the ui; I can add the others but we aren't using them at this point